extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -110,8 +113,15 @@ export class DbConfigStore extends DisposableObject {
     }
 
     const config: DbConfig = cloneDbConfig(this.config);
-    config.databases.remote.repositories.push(repoNwo);
-
+    if (parentList) {
+      config.databases.remote.repositoryLists.forEach((list) => {
```

Comment:
Shall we also check whether the parent list exists at all and throw an error if it doesn't?

Reply:
Hm, in which case could it not exist?.. We can check for it though.

Reply:
It's one of those things that _should never_ happen.